feature-libs/commerce-quotes/components/commerce-quotes-actions-by-role/commerce-quotes-actions-by-role.module.ts

```diff
@@ -0,0 +1,47 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { CQConfig } from '@spartacus/commerce-quotes/core';
+import { QuoteAction } from '@spartacus/commerce-quotes/root';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import { CommerceQuotesActionsByRoleComponent } from './commerce-quotes-actions-by-role.component';
+
+@NgModule({
+  imports: [CommonModule, I18nModule],
+  providers: [
+    provideDefaultConfig(<CQConfig>{
+      commerceQuotes: {
+        primaryActions: [
+          QuoteAction.APPROVE,
+          QuoteAction.CHECKOUT,
+          QuoteAction.SUBMIT,
+        ],
+        actionsOrderByState: {
+          BUYER_DRAFT: [QuoteAction.CANCEL, QuoteAction.SUBMIT],
+          CANCELLED: [QuoteAction.REQUOTE],
+          SELLER_REQUEST: [QuoteAction.EDIT, QuoteAction.SUBMIT],
+          BUYER_OFFER: [
+            QuoteAction.CANCEL,
+            QuoteAction.EDIT,
+            QuoteAction.CHECKOUT,
+          ],
+        },
```

Comment:
What do you think about wrapping this into an object to make it more context-precise, because now there will be two separate fields relating to the same configuration? So my proposition would be to wrap it into "actions". Thoughts?

Reply:
Consider it done!